background_randomizer.js

```diff
@@ -0,0 +1,60 @@
+var time_elapsed = 0,
+    colors_used = [],
+    time_per_interval = 1000,
+    total_time = 10000,
+    testing = false,
+    found_repeat = 0,
+    found_unique = 0;
+
+if (testing) {
+    time_per_interval = 100; // Speed up when testing
+}
+
+function calc_color() {
+    var r, g, b, rgb;
+
+    r = parseInt(Math.random() * 255);
```

Comment:
Ah no it will not "up to but not including 1 (exclusive)" ( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random ). 
```Math.floor(Math.random() * (255 + 1));```